Dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1
+   
+FROM python:3.9.6
+WORKDIR /app
+COPY requirements.txt requirements.txt
```

Comment:
Wouldn't this be covered by line 7?

Reply:
If that's the case, then would I move line 7 to line 5 so that line 6 knows what to run?